homeassistant/components/knx/websocket.py

```diff
@@ -0,0 +1,135 @@
+"""KNX Websocket API."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from typing import Final
+
+import voluptuous as vol
+from xknx.telegram import Telegram, TelegramDirection
+from xknx.telegram.apci import GroupValueRead, GroupValueResponse, GroupValueWrite
+from xknx_custom_panel import get_knx_ui
+
+from homeassistant.components import panel_custom, websocket_api
+from homeassistant.core import HomeAssistant, callback
+
+from .const import (
+    DOMAIN,
+    AsyncMessageCallbackType,
+    KNXBusMonitorMessage,
+    MessageCallbackType,
+)
+
+URL_BASE: Final = "/static_knx"
+
+
+async def register_panel(hass: HomeAssistant) -> None:
+    """Register the KNX Panel and Websocket API."""
+    websocket_api.async_register_command(hass, ws_info)
+    websocket_api.async_register_command(hass, ws_subscribe_telegram)
+
+    if DOMAIN not in hass.data.get("frontend_panels", {}):
+        hass.http.register_static_path(URL_BASE, get_knx_ui())
+        await panel_custom.async_register_panel(
+            hass=hass,
+            frontend_url_path=DOMAIN,
+            webcomponent_name="knx-panel",
+            sidebar_title=DOMAIN.upper(),
+            sidebar_icon="mdi:earth",
+            module_url=f"{URL_BASE}",
```

Comment:
This should point at the JavaScript, but it now points ` /static_knx` ? It should be the same URL as registered for trhe static path?

Reply:
Yep, still need to fix this. Currently I just map `/static_knx` to the final JS: https://github.com/XKNX/custom-panel/blob/main/xknx_custom_panel/__init__.py#L7

Which is a bit weird - so will change that. I also have to add the hashing still.